webapp/nuxt.config.js

```diff
@@ -123,7 +123,7 @@ export default {
   ],
 
   router: {
-    middleware: ['authenticated', 'termsAndConditions'],
+    middleware: ['authenticated', 'termsAndConditions', 'searchHashtag'],
```

Comment:
```suggestion
    middleware: ['authenticated', 'termsAndConditions'],
```

You are adding a middleware for every route here. I would say the feature does not justify the added complexity :neutral_face: